src/App.tsx

```diff
@@ -1,10 +1,60 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import cn from 'classnames';
+
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { FilterStatus } from './types/FilterStatus';
 
-const USER_ID = 0;
+const USER_ID = 10999;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | null>(null);
+  const [isError, setIsError] = useState(false);
+  const [filterStatus, setFilterStatus] = useState(FilterStatus.ALL);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then((todosFromServer) => {
+        setTodos(todosFromServer);
+      })
+      .catch((error) => {
+        setIsError(true);
+        throw new Error(error.message);
+      });
+  }, []);
+
+  useEffect(() => {
+    if (isError) {
+      const timeout = setTimeout(() => {
+        setIsError(false);
+      }, 3000);
+
+      return () => clearTimeout(timeout);
+    }
+
+    return () => {};
```

Comment:
looks redundant

Reply:
yeah, but without this return, a linter error occurs